packages/@okta/vuepress-theme-default/assets/css/okta/pages/code.scss

```diff
@@ -8,3 +8,16 @@ ul.code-list {
   padding-left: 0px;
 }
 
+
+.language-content a:not([class="header-anchor header-link"]) {
```

Comment:
CC: @edburyenegren-okta

Reply:
Instead of using `:not`, could we attach a class to the desired stylings instead?

Or, if the purpose is to target "normal" links, maybe `a:not([class="*"])`?

Reply:
@edburyenegren-okta I would love to do this, but for now, I can not wrap arbitrary elements through Markdown.  I am looking at solution to be able to do this, but for now, using `:not` in the css is necessary to not include the deep link anchor for headers.  `.language-content` is only used on code pages, so it will not affect anything else.